.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+
+python:
+  - "2.6"
+  - "2.7"
+
+script:
+  - python runtests.py
+
+install:
+- pip install django-dirtyfields
```

Comment:
I think at the end, we just need to install dirtyfields requirements, and launch tests. You are right, it's fully wrong to install via pip, we want to test the latest one.